ui/App/SettingsScreen.svelte

```diff
@@ -5,9 +5,28 @@
  with Radicle Linking Exception. For full terms see the included
  LICENSE file.
 -->
-<script lang="typescript">
+<script lang="typescript" context="module">
   import * as svelteStore from "svelte/store";
+  import persistentStore from "svelte-persistent-store/dist";
+
+  export type PrimaryColor = "blue" | "pink" | "orange";
+  export const primaryColorStore = persistentStore.local.writable<PrimaryColor>(
+    "radicle.settings.primaryColor",
+    "blue"
+  );
+
+  export const updatePrimaryColor = (event: CustomEvent): PrimaryColor => {
+    primaryColorStore.set(event.detail);
+  };
 
+  export const primaryColorOptions = [
```

Comment:
We could add an explicit type here to avoid adding new items that are not supported by accident:
```suggestion
  export const primaryColorOptions: { title: string; value: PrimaryColor }[] = [
```